Components/CreateTrip/CreateTrip.js

```diff
@@ -1,132 +1,74 @@
 import React, { useState } from 'react';
-import { Image, StyleSheet, Text, View, TextInput, TouchableOpacity } from 'react-native';
+import { StyleSheet, Text, View, TextInput, TouchableOpacity, ScrollView } from 'react-native';
 import { postNewTrip } from '../../apiCalls';
 
-
 export const CreateTrip = ({ navigation }) => {
   return (
-    <View style={styles.container}>
-      <View style={styles.header}>
-        <Image
-          style={styles.image}
-          source={{uri: 'https://img.icons8.com/officel/40/000000/globe.png'}}
-        />
-        <Text style={{fontSize: 20, marginLeft: 33}}>
-          New Trip
-        </Text>
-        <Text style={{color: '#96CDEC', marginRight: 1}}>
-          placehold
-        </Text>
-      </View>
-      <View>
-        <Text>Trip name:</Text>
-        <TextInput style={styles.input}
-          placeholder='e.g. Spring break'
-        />
+    <ScrollView
+      contentContainerStyle={styles.container}
+      keyboardShouldPersistTaps='handled'
+    >
+      <Text style={styles.label}>Name</Text>
+      <View style={styles.inputContainer}>
+        <TextInput style={styles.input} />
       </View>
```

Comment:
Still getting used to the new toys that come with React-Native. Good training for my familiarity to see these built-in aspects (contentContainerStyle & keyboardShouldPersistTaps).

Thanks, Scott!